model.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+
+const UserSchema = new mongoose.Schema({
+  username: {
```

Comment:
You should make some of these properties `required` and `username` should also be marked as `unique`